### DISCUSSION TITLE
Cleaning up livecheck --all

### DISCUSSION THREAD

**Discussion by p-linnane** [Homebrew/discussions#3360] [Tap maintenance and Homebrew development] [answered]
Hello,

Lately I've been running `brew livecheck --all --cask --newer-only` and trying to resolve any failures. There are a number of casks that cannot be updated using `brew bump-cask-pr` for various reasons such as livecheck failure or changes to application URLs. I wanted to see if anyone would like to collaborate on working together toward resolving these existing cask issues. Please reply if you're interested and we can discuss the best way to coordinate.

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
Many people have been fixing livecheck issues. Just make a pull request to the repo where you found the issue, no coordination required.
